goci-interfaces/goci-ui/src/main/resources/static/js/resultexpansion.js

```diff
@@ -330,6 +330,9 @@ function addResults(data, expand, id) {
                         var geneDescription = '' // initializing empty description
 
                         var descriptionElements = descriptionTruncated.split("|");
+                        if (! descriptionElements[0]){
+                            descriptionElements[0] = "No description available.";
+                        }
```

Comment:
In the future, will this default text be added to the Solr document directly?

Reply:
Yes.